lib/start/src/bottom_stuff.dart

```diff
@@ -68,12 +78,13 @@ class BottomStuff extends HookWidget {
               onPressed: LoginLabels.goto(target),
               child: SizedBox(
                 width: double.infinity,
-                child: Text(text, style: spaced, textAlign: TextAlign.center),
+                child: Text(text,
+                    style: spaced.copyWith(fontSize: fontSize), textAlign: TextAlign.center),
```

Comment:
Since I didn't override the `.copyWith()` method, using `fontSize` here is the right way to do it :)

Adding a trailing comma (i.e. `TextAlign.center,),`) would help the auto-formatter make this a bit more readable.